composer.json

```diff
@@ -52,7 +53,7 @@
         "drupal/core-recommended": "^9.1",
         "drupal/date_popup": "^1.1",
         "drupal/diff": "^1.0",
-        "drupal/entity_print": "^2.2",
+        "drupal/entity_print": "^2.13",
```

Comment:
Do we really need to (potentially) downgrade entity print to make this work?

Reply:
No. That is changed